pages/ksphere/kommander/latest/introducing-kommander/index.md

```diff
@@ -16,9 +16,9 @@ Kommander was developed to address the broad issues caused by Kubernetes “clus
 | **Features**                                     | **Benefits**                                                 |
 | ------------------------------------------------ | ------------------------------------------------------------ |
 | **Operations Dashboard**                         | Provides instant visibility and operational efficiency into Kubernetes cluster landscape, helping an organization understand resources and utilization. |
-| **Centralized Monitoring, Logging and Alerting** | Enables central monitoring and alerting of issues arising in operational and application infrastructure and deep cross-cluster logging for easier troubleshooting. As issues are detected, they are resolved before they escalate, saving valuable time. |
-| **Cluster Configuration Manager**                | Simplifies and delivers consistent configuration for services and cross cluster operations. |
-| **Services Catalog**                             | Push button catalog of current cloud native services. This helps organizations quickly deploy services to multiple clusters. Governance-based access to data services ensures roles and responsibilities are maintained. |
+| [**Centralized Monitoring, Logging and Alerting**](/ksphere/kommander/latest/centralized-monitoring)  | Enables central monitoring and alerting of issues arising in operational and application infrastructure and deep cross-cluster logging for easier troubleshooting. As issues are detected, they are resolved before they escalate, saving valuable time. |
+| [**Cluster Configuration Manager**](/ksphere/kommander/latest/clusters)                | Simplifies and delivers consistent configuration for services and cross cluster operations. |
+| [**Services Catalog**](/ksphere/kommander/latest/clusters/#platform-services)                             | Push button catalog of current cloud native services. This helps organizations quickly deploy services to multiple clusters. Governance-based access to data services ensures roles and responsibilities are maintained. |
```

Comment:
All of your PRs are building off the same commit, can you separate that out so that each PR is focused specifically on the proposed changes?